susemanager-utils/susemanager-sls/salt/bootstrap/init.sls

```diff
@@ -26,14 +26,31 @@ mgr_server_localhost_alias_absent:
 {%- else %}
 {% set bootstrap_repo_url = 'https://' ~ salt['pillar.get']('mgr_server') ~ '/pub/repositories/' ~ os_base ~ '/' ~ grains['osrelease'] ~ '/0/bootstrap/' %}
 {%- endif %}
+
 {%- elif grains['os_family'] == 'RedHat' %}
-{%- if salt['file.file_exists' ]('/etc/oracle-release') %}
+{%- if salt['file.file_exists']('/etc/oracle-release') %}
 {% set bootstrap_repo_url = 'https://' ~ salt['pillar.get']('mgr_server') ~ '/pub/repositories/oracle/' ~ grains['osmajorrelease'] ~ '/bootstrap/' %}
-{%- elif salt['file.file_exists' ]('/etc/redhat-release') %}
+
+{%- elif salt['file.file_exists']('/usr/share/doc/sles_es-release') %}
+{% set bootstrap_repo_url = 'https://' ~ salt['pillar.get']('mgr_server') ~ '/pub/repositories/res/' ~ grains['osmajorrelease'] ~ '/bootstrap/' %}
+
+{%- elif salt['file.file_exists']('/etc/centos-release') %}
+{# We try CentOS bootstrap repository first. Failback to RES #}
+{% set bootstrap_repo_url = 'https://' ~ salt['pillar.get']('mgr_server') ~ '/pub/repositories/centos/' ~ grains['osmajorrelease'] ~ '/bootstrap/' %}
+{% set bootstrap_repo_request = salt['http.query'](bootstrap_repo_url + 'repodata/repomd.xml', status=True, verify_ssl=False) %}
+{%- if bootstrap_repo_request['status'] == 901 %}
```

Comment:
`901`? :-?

Reply:
Special SSL error. TLS 1.2 not available ... you know this beast :-)

Reply:
This same code is reused in this SLS file. It actually fixes some scenario we already got a bug report.

I think the status 901 is expected in cases you have a captive portal or something like that.

Reply:
ACK :-)